lib/workers/repository/init/merge.ts

```diff
@@ -245,6 +245,16 @@ export async function mergeRenovateConfig(
     logger.debug('Found npmrc in decrypted config - setting');
     npmApi.setNpmrc(decryptedConfig.npmrc);
   }
+
+  // Provide insight into the repository config, resolving everything but internal Renovate presets
+  // This allows users to understand the fully resolved configuration, including any `github>`, `local>`, etc presets, but excluding anything that's internal to Renovate (which can be verbose)
+  const resolvedConfigWithoutInternalPresets =
+    await presets.resolveConfigPresets(decryptedConfig, config, [], [], false);
```

Comment:
Should we only do it if this log line is enabled? It's not that much extra processing (as we cache) so maybe it's OK